apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConfigServiceLocator.java

```diff
@@ -194,6 +200,11 @@ private synchronized void updateConfigServices() {
     String url = assembleMetaServiceUrl();
 
     HttpRequest request = new HttpRequest(url);
+    // speed up http request
+    int mixTimeout = Math.min(m_configUtil.getConnectTimeout(), m_configUtil.getReadTimeout());
+    request.setConnectTimeout(mixTimeout);
+    request.setReadTimeout(mixTimeout);
```

Comment:
Use min timeout here.

Reply:
The connection time and read timeout serve different functions, so using the minimum of the two may not be ideal.